torch/_inductor/simple_fsdp/utils.py

```diff
@@ -130,3 +136,163 @@ def get_node_type(node: "scheduler.BaseSchedulerNode") -> NodeType:
             return NodeType.COMPUTE
 
     return _get_ir_node_type(node.node)
+
+
+def _get_benchmark_runtime(node) -> List[float]:
+    """
+    Returns estimated op runtime in nanoseconds (ns)
+    """
+    # Communication kernel benchmark
+    if is_collective(node.node):
+        return [0, 0]
```

Comment:
pls comment something like: no need to estimate comm time at the single-node level, since we will always do dynamic comm time estimation inside the greedy algorithm